src/behaviors/Column.js

```diff
@@ -24,25 +24,35 @@ var deprecated = require('../lib/deprecated');
  *    -2   | Tree (drill-down) column
  */
 function Column(behavior, options) {
-    var index, fields;
+    var index, schema;
 
     this.behavior = behavior;
     this.dataModel = behavior.dataModel;
-    fields = this.dataModel.getFields();
+
+    schema = this.behavior.dataModel.schema;
 
     switch (typeof options) {
         case 'number':
             index = options;
             options = {};
             break;
         case 'string':
-            index = fields.indexOf(options);
+            index = getIndexFromName(options);
             options = {};
             break;
         case 'object':
             index = options.index !== undefined
                 ? options.index
-                : fields.indexOf(options.name);
+                : getIndexFromName(options.name);
+    }
+
+    function getIndexFromName(name) {
+        var index;
+        schema.find(function(columnSchema, i) {
```

Comment:
Let's use `findIndex` here. Because it is newer than Chrome 40, add it's [polyfill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex#Polyfill) to lib/polyfills.js.